helm/charts/kratos/values.yaml

```diff
@@ -56,29 +56,31 @@ secret:
 ingress:
   admin:
     enabled: false
+    className: ""
```

Comment:
Will it be sufficient/work? Reading https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#setting-a-default-ingressclass tells it will work if one has annotated IngressClass CR but what if hasn't ?

Reply:
The resulting manifests will be correct and valid from helm/k8s viewpoint. Whether they will work OOTB? I don't think so, not expect them to. Users will need to supply proper values, and they can do that, as the required config is exposed via values

Reply:
General comment: charts should work in minimal setup OOTB after doing `helm install`, if there are any specific **required** values to be provided in order to work or other prerequisites, They have to be described as well as the behaviour. I would put that desc in chart readme.

Reply:
Yes, this is still valid. Please remember that ingress objects are optional, and disabled by default. Enabling them requires the user to provide additional input/configuration. The checks I've added in this PR ensure that:
- charts work OOTB - they can be installed, have to validation errors, and the pods are in a running state
- ingress objects are valid and configurable

If we'd like to check if the ingress objects work, we would need to install an ingress controller and modify the hosts file to check connectivity. But this is imho a lot of overhead and testing if k8s features works, rather then ours

Reply:
What I meant is add an info for Ingress what is required to make it work OOTB if one enables it